src/lib/prisma.js

```diff
@@ -1,5 +1,24 @@
 import Prisma from "@prisma/client";
+import metrics from "../metrics.js";
 
-let prisma = new Prisma.PrismaClient();
+let prisma = new Prisma.PrismaClient().$extends({
```

Comment:
whoa

Reply:
I mean - I do get what you're doing here, I do think it deserves a high level comment like "adding hook into all query operations to add telemetry" or something